hw/xbox/smbus_xbox_smc.c

```diff
@@ -55,10 +55,11 @@
 #define SMC_REG_AVPACK              0x04
 #define     SMC_REG_AVPACK_SCART        0x00
 #define     SMC_REG_AVPACK_HDTV         0x01
-#define     SMC_REG_AVPACK_VGA_SOG      0x02
+#define     SMC_REG_AVPACK_VGA          0x02
+#define     SMC_REG_AVPACK_RFU          0x03
 #define     SMC_REG_AVPACK_SVIDEO       0x04
 #define     SMC_REG_AVPACK_COMPOSITE    0x06
```

Comment:
Is called `_STANDARD` in kernel (and also as your "avpack" option, creating an inconsistency).

I wonder if we should make an exception here, and call it "composite" (including "avpack" option) across our codebase as to not confuse the reader with the MS name. "standard" is very vague - it was fitting at the time, but by today's standards, we'd consider "standard"=HDMI.

Reply:
I'd leave it as composite because it describes it better. Do you know other places where it needs to be fixed up for consistence?

Reply:
Only your "avpack" option name probably.